sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1115,15 +1115,18 @@ def get_loopback_capability(self):
         loopback_capability['media_side_output_loopback_supported'] = bool((allowed_loopback_result >> 0) & 0x1)
         return loopback_capability
 
-    def set_loopback_mode(self, loopback_mode):
+    def set_loopback_mode(self, loopback_mode, lane_mask = 0xff):
         '''
         This function sets the module loopback mode.
-        Loopback mode has to be one of the five:
+        loopback_mode: Loopback mode has to be one of the five:
         1. "none" (default)
         2. "host-side-input"
         3. "host-side-output"
         4. "media-side-input"
         5. "media-side-output"
+        lane_mask: A bitmask representing which lanes to apply the loopback mode to.
+        The default value of 0xFF indicates that the mode should be applied to all lanes.
+
         The function will look at 13h:128 to check advertized loopback capabilities.
         Return True if the provision succeeds, False if it fails
         '''
```

Comment:
1. Sure! we'd better to reject the request if simultaneous host media loopback mode is not supported. 
2. To clear specific lanes corresponding to the logical port for the **none** mode, we will need to have two lane masks (**host_lane_mask** and **media_lane_mask** for host and media individually) because the host and media lane masks might not be identical (e.g., host_lane_mask = 0x3, media_lane_mask = 0x1).
3. If per-lane loopback mode is not supported, should we reject the request or configure the entire physical port instead?
4. Do you think it is useful to log some message for the reject cases?

Reply:
@xinyulin
For `2.`, we can expect `loopback_mode` to include the type of loopback to be cleared i.e. have `host-side-input-none`, `host-side-output-none` etc. In this case, the caller can pass the lane_mask accordingly.
For `3.`, I think we should configure entire physical port and display a message. @prgeor Can you please confirm the expected behavior in this case?
For `4.`, Yes, I think it is helpful to log messages in reject cases.

Reply:
@mihirpat1 Thanks for the comments. I've updated it accordingly. Let's check if Prince has any comments on the changes.

Reply:
@xinyulin I discussed with @prgeor regarding `3.` and concluded to reject the config if per lane loopback is not supported (i.e. we should not configure unrelated lanes when loopback is intended to configure specific lanes).

Reply:
@mihirpat1 Thanks for the confirmation. It has been updated!